src/pages/index.js

```diff
@@ -3,10 +3,73 @@ import clsx from 'clsx';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
-import HomepageFeatures from '@site/src/components/HomepageFeatures';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Translate from '@docusaurus/Translate';
 
 import styles from './index.module.css';
 
+const FeatureList = [
+  {
+    title: 'Easy Installations',
+    description: (
+      <>
+        Komiser was designed to be light-weight and easily installed so 
+        you can  start getting cloud insights as quickly as possible.
+      </>
+    ),
+    imageUrl: 'img/easy-installations.png'
+  },
+  {
+    title: 'Focus on What Matters',
+    description: (
+      <>
+        Komiser enables you to have a clear view into your cloud account, gives helpful advice to reduce the cost and secure your environment.
+      </>
+    ),
+    imageUrl: 'img/focus-on-what-matters.png'
+  },
+  {
+    title: 'The Future of DevOps',
+    description: (
+      <>
+        Komiser is the first step in our plan to create the future generation of Developers and DevOps engineers. The future of DevOps can only be built with the understanding of what we actually have in our cloud accounts, and Komiser is here to help with exactly that.
```

Comment:
Just rephrased to keep it short and aligned "The future of DevOps start with understanding of what we actually have in our cloud accounts, and Komiser can you help discover your resources.", feel free to tweak the wording ofc

![Screenshot 2023-01-19 at 12 44 19](https://user-images.githubusercontent.com/10320205/213434084-eebad91d-b062-4db0-9263-92c61c8a5585.png)

Reply:
I tweaked it a little to "The future of DevOps starts with knowing exactly what we have in our cloud accounts, and Komiser can you help discover every last one of your resources."
Cool?